packs/spell-effects/spell-effect-elephant-form.json

```diff
@@ -4,7 +4,7 @@
+            "value": "<p>Granted by @UUID[Compendium.pf2e.spells-srd.Item.Elephant Form]</p>\n<p>You transform into a Large battle form, specifically that of a elephant or similar creature, such as a mammoth, mastodon, or anancus. This spell has the effects of @UUID[Compendium.pf2e.spells-srd.Item.Dinosaur Form], except you gain the following specific abilities instead of the form-specific abilities of dinosaur form.</p><ul><li>Speed 40 feet;</li><li><strong>Melee</strong> <span class=\"action-glyph\">1</span> foot, <strong>Damage</strong> 2d8 bludgeoning.</li><li><strong>Melee</strong> <span class=\"action-glyph\">1</span> trunk (reach 10 feet), <strong>Effect</strong> grabbing trunk;</li><li><strong>Melee</strong> <span class=\"action-glyph\">1</span> tusk (reach 10 feet), <strong>Damage</strong> 2d6 piercing.</li></ul><p><strong>Grabbing Trunk</strong> A creature up to two sizes smaller that's hit by your trunk is @UUID[Compendium.pf2e.conditionitems.Item.Grabbed]. If you succeed at your @UUID[Compendium.pf2e.actionspf2e.Item.Grapple] check against a creature grabbed by your trunk, you can reposition the creature to another space within your trunk's reach.</p><hr /><p><strong>Heightened (5th)</strong> Your elephant form is Huge; your trunk and tusk attacks have 15-foot reach, and your foot attack has 10-foot reach. You instead gain 20 temporary HP, an attack modifier of +18, a damage bonus of +6, double the damage dice, and Athletics +21.</p>\n<p><strong>Heightened (7th)</strong> Your elephant form is Gargantuan; your trunk and tusk attacks have 20-foot reach, and your foot attack has 15-foot reach. Your AC is 21 + your level and you instead gain 25 temporary HP, an attack modifier of +25, a damage bonus of +15, double the damage dice, and Athletics +25.</p>"
```

Comment:
```suggestion
            "value": "<p>Granted by @UUID[Compendium.pf2e.spells-srd.Item.Elephant Form]</p>\n<p>You transform into a Large battle form, specifically that of a elephant or similar creature, such as a mammoth, mastodon, or anancus. This spell has the effects of <em>dinosaur form</em>, except you gain the following specific abilities instead of the form-specific abilities of <em>dinosaur form</em>.</p><ul><li>Speed 40 feet;</li><li><strong>Melee</strong> <span class=\"action-glyph\">1</span> foot, <strong>Damage</strong> 2d8 bludgeoning.</li><li><strong>Melee</strong> <span class=\"action-glyph\">1</span> trunk (reach 10 feet), <strong>Effect</strong> grabbing trunk;</li><li><strong>Melee</strong> <span class=\"action-glyph\">1</span> tusk (reach 10 feet), <strong>Damage</strong> 2d6 piercing.</li></ul><p><strong>Grabbing Trunk</strong> A creature up to two sizes smaller that's hit by your trunk is grabbed. If you succeed at your Grapple check against a creature grabbed by your trunk, you can reposition the creature to another space within your trunk's reach.</p><hr /><p><strong>Heightened (5th)</strong> Your elephant form is Huge; your trunk and tusk attacks have 15-foot reach, and your foot attack has 10-foot reach. You instead gain 20 temporary HP, an attack modifier of +18, a damage bonus of +6, double the damage dice, and Athletics +21.</p>\n<p><strong>Heightened (7th)</strong> Your elephant form is Gargantuan; your trunk and tusk attacks have 20-foot reach, and your foot attack has 15-foot reach. Your AC is 21 + your level and you instead gain 25 temporary HP, an attack modifier of +25, a damage bonus of +15, double the damage dice, and Athletics +25.</p>"
```